### DISCUSSION TITLE
These days whenever I open Cider I need to log in again using my AppleID

### DISCUSSION THREAD

**Discussion by MarinatedEgg** [ciderapp/Cider#869] [General]
Is there something wrong？

**Comment by rlaphoenix** [comment]
Were you in Europe? It seems recently their API used in the App broke in the EU, causing stuff to break if various files were not cached.
I've had this issue too, and it seems as of 4 days ago from this post they seemed to have moved some of these checks to before the UI loads and before it asks for auth, so if that does fail to load those files, it will tell you and will not try to have you authenticate anymore.

I'm also being told in the Discord that they are working on fixing the EU API as it seems they only just now found out its broken in the EU.

**Comment by maikirakiwi** [comment]
This problem only occurred when the API was deploying to our new location in Switzerland for the first time, and resolved within 24 hours. (If you are in or near Europe)
